Header header
string type
string uuid
int16 majorVal
int16 minorVal
int16 rssi
string macAddress
string url
string namespace
string instance
